spec/rubocop/cop/style/extra_spacing_spec.rb

```diff
@@ -247,7 +247,8 @@
     end
 
     it 'does not register an offense if assignments are aligned' do
-      inspect_source(cop, ['a   = 1',
+      inspect_source(cop, ['# comment',
```

Comment:
I think this should probably be a separate example to illustrate what we're testing in a clearer manner.